src/content/docs/style-guide/images/image-annotations.mdx

```diff
@@ -0,0 +1,101 @@
+
+If you need to use image annotations:
+
+* Be concise.
+* Make sure annotations are easy to distinguish from UI text. For example, you may need to offset your boxes so they don't look like UI elements.
+* Annotations aren't translated, so use them sparingly.
+* Don't use an annotation at the top that acts like a title. Use a caption.
+* Use our [New Relic SnagIt theme](https://github.com/newrelic/newrelic-doc-tools) (link only available to New Relic employees) to keep your annotations consistent.
+
+## Install the New Relic SnagIt theme [#install-theme]
+
+  Follow these steps:
+    1. Ensure you're running the latest version of SnagIt.
+    2. In GitHub Desktop, clone the [repo](https://github.com/newrelic/newrelic-doc-tools) locally.
+    3. Open SnagIt and select any tool to open the right nav.
+    4. Under the **Quick Styles** pane in the right nav, select the **Theme** gear icon, then **Import**.
+
+## Callout boxes [#callout-boxes]
+
+Use callouts to highlight an element within a screenshot. If possible, don't add text to screenshots, that's what our docs are for. Callout boxes should not obstruct any of the elements they're calling out. For example, a callout box around a UI button shouldn't cover any part of the button. If possible, insert a two-pixel clearance between the UI element and the callout box.
+
+This is the format for adding callout boxes:
+  * Shape: square or rectangle, transparent fill, no border-radius.
+  * Border color: #00CE7C
+  * Line width: 5px
+  * Opacity: 100%
+  * Line style: solid
+  * No shadows
+
+<img
+  title="Example of a callout box"
+  alt="Example of a callout box"
+  src={styleguideExampleCalloutBox}
+/>
+
+<figcaption>
+  This is an example of a callout box.
+</figcaption>
+
+## Callout text [#callout-text]
+
+Screenshots shouldn't require a text callout 99 percent of the time; `<figcaptions>` and written content before or after the image should be sufficient to explain the screenshot. However, if a text box is required, it shouldn't overlap any essential UI text or elements.
+
+If you've to add text to a screenshot, follow this format:
+
+  * Shape: square or rectangle, fill color #00CE7C.
+  * Border color: #00CE7C
+  * Line width: 5px
+  * Opacity: 100%
+  * Line style: solid
+  * No shadows
+  * Font: Arial bold #FFFFFF, size 24px, left justified
+
+<img
+  title="dashboards_api_entity_section.png"
+  alt="dashboards_api_entity_section.png"
+  src={apisanddataNerdGraphDashboardEntity}
+/>
+
+<figcaption>
+  An example of a callout with text.
+</figcaption>
+
+## Arrows [#arrows]
+
+Use a single-headed arrow or an arrow with arrowheads on each end in your screenshot when you want to reference elements within the UI.
+
+If you use an arrow, try to not overlap any essential UI text or elements.
+
+This is the format for adding an arrow:
+  * Opacity: 100%
+  * Line width: 7px
```

Comment:
@nbaenam I just realized that SnagIt uses "pt" instead of "px." They are slightly different, so will that affect the value you want? in this case, the value should be "7pt"